src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Nunito:wght@300&family=Open+Sans:wght@300&family=Poppins:wght@200&display=swap");
+
+.forecast-summary {
+  padding: 10px 16px;
+  text-align: center;
+  border: 1px solid rgb(210, 210, 210);
+  margin-top: 1rem;
+  background-color: rgb(247, 253, 255);
+  min-width: 14rem;
+}
+
+.forecast-summary__icon {
+  font-size: 3rem;
+  font-weight: bold;
+  margin-top: 2rem;
+}
+
+.forecast-summary__date {
+  margin: 0.5rem;
+  font-size: 1.5rem;
+  font-family: "Poppins", sans-serif;
+}
+
+.forecast-summary__temperature {
+  margin: 0.5rem;
+  font-size: 1.2rem;
+}
+
+.forecast-summary__description {
+  margin: 0.5rem;
+  font-size: 1.2rem;
+}
```

Comment:
This could be tidied up a bit to reduce repetition:
```
.forecast-summary__temperature,
.forecast-summary__description {
  margin: 0.5rem;
  font-size: 1.2rem;
}

.forecast-summary__date {
  font-size: 1.5rem;
  font-family: "Poppins", sans-serif;
}
```